charts/geoserver/v0.3.1/templates/deployment.yaml

```diff
@@ -37,8 +37,6 @@ spec:
         {{- (tpl . $) | nindent 8 }}
         {{- end }}
     spec:
-      securityContext:
```

Comment:
Doesn't this work or it was already added ?

Reply:
These were merged directly. I'll create a pull request with a new Helm chart on the main branch. This will trigger the action to generate a new Helm chart.